### DISCUSSION TITLE
Lab Assignment #3

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#14] [Assignments]
## Submission
Paste your GitHub pages URL for `Week_04` as a comment below to submit your  [lab assignment #3](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_04/Lab/assignment.md).

e.g. https://akochaphum.github.io/211A-ASIAAM-191A-Assignments/Week_04/index.html

**Comment by luigimagdamit** [comment]
https://luigimagdamitucla.github.io/211A-191A-Lab-Assignments/week4/

**Reply by albertkun** [reply]
@LuigiMagdamitUCLA i added a point! see if you can find it! :) also, i think the icon anchoring is a bit off!

**Reply by luigimagdamit** [reply]
UHHH SAN JOSE????? Yes, you are right about the anchoring. Thanks!

**Reply by albertkun** [reply]
@LuigiMagdamitUCLA no problem! glad it helped!!

**Comment by abi-200** [comment]
https://abi-200.github.io/191a-labs/week4/

**Reply by albertkun** [reply]
@abi-200 see if you notice my point! :)

**Reply by abi-200** [reply]
Yes Las Vegas!

**Reply by albertkun** [reply]
@abi-200 correct!

**Comment by edmondywen** [comment]
https://edmondywen.github.io/asiaam-191a/week4/index.html

**Reply by albertkun** [reply]
@edmondywen hmm! i don't think i have permission to see your survey!
![image](https://user-images.githubusercontent.com/8574425/126432398-596a5bf7-b907-4521-8a84-926e3a139ef4.png)

**Reply by edmondywen** [reply]
I think I fixed it. hopefully it works!

**Reply by albertkun** [reply]
@edmondywen it does! thank you!

**Comment by cynthinee** [comment]
https://cynthinee.github.io/2021A-191A-Lab-Assignments/week4/

**Reply by albertkun** [reply]
@cynthinee I added a point! See if you can find it! :)

**Reply by cynthinee** [reply]
@albertkun Fresno!🎉

**Reply by albertkun** [reply]
@cynthinee yep!! good job!

**Comment by ramirezirvin** [comment]
https://ramirezirvin.github.io/191A-lab-assingments/week4/

**Comment by hrfrahn** [comment]
https://hrfrahn.github.io/191A-Labs/week4/

**Comment by eandrusenki** [comment]
https://eandrusenki.github.io/AsAmS-191A-Assignments/week4/

**Comment by perezsara59** [comment]
https://perezsara59.github.io/211A-191A-Assignments/week4/

**Comment by emilysuh** [comment]
https://emilysuh.github.io/211A-191A-Lab-Assignment/week4/

**Comment by ronnie-blanche** [comment]
http://10.1.10.179:61302/week%204/
